refinery/core/utils.py

```diff
@@ -21,6 +21,9 @@
 import core
 import data_set_manager
 
+from core.search_indexes import DataSetIndex
+from data_set_manager.search_indexes import NodeIndex
```

Comment:
`core.utils` is also the file that introduces a nifty spiderweb of imports. 

There is not much we can do atm without some major refactoring/tackling of technical debt